sdk/monitor/azure-monitor-ingestion/README.md

```diff
@@ -0,0 +1,179 @@
+- A [Data Collection Endpoint][data_collection_endpoint]
+- A [Data Collection Rule][data_collection_rule]
+- A [Log Analytics workspace][log_analytics_workspace]
+
+### Include the package
+
+[//]: # ({x-version-update-start;com.azure:azure-monitor-ingestion;current})
+```xml
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-monitor-ingestion</artifactId>
+    <version>1.0.0-beta.1</version>
+</dependency>
+```
+
+### Create the client
+
+An authenticated client is required to upload logs to Azure Monitor. The library includes both synchronous and asynchronous forms of the clients. To authenticate, 
+the following examples use `DefaultAzureCredentialBuilder` from the [com.azure:azure-identity](https://search.maven.org/artifact/com.azure/azure-identity) package.
+
+#### Authenticating using Azure Active Directory
+You can authenticate with Azure Active Directory using the [Azure Identity library][azure_identity].
+To use the [DefaultAzureCredential][DefaultAzureCredential] provider shown below, or other credential providers provided with the Azure SDK, please include the `azure-identity` package:
+[//]: # ({x-version-update-start;com.azure:azure-identity;dependency})
+```xml
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-identity</artifactId>
+    <version>1.5.2</version>
+</dependency>
+```
+Set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables: AZURE_CLIENT_ID, AZURE_TENANT_ID, AZURE_CLIENT_SECRET.
+
+#### Synchronous Logs Ingestion client
+
+```java readme-sample-createLogsIngestionClient
+DefaultAzureCredential tokenCredential = new DefaultAzureCredentialBuilder().build();
+
+LogsIngestionClient client = new LogsIngestionClientBuilder()
+        .endpoint("<data-collection-endpoint>")
+        .credential(tokenCredential)
+        .buildClient();
+```
+
+#### Asynchronous Logs Ingestion client
+
+```java readme-sample-createLogsIngestionAsyncClient
+DefaultAzureCredential tokenCredential = new DefaultAzureCredentialBuilder().build();
+
+LogsIngestionAsyncClient asyncClient = new LogsIngestionClientBuilder()
+        .endpoint("<data-collection-endpoint>")
+        .credential(tokenCredential)
+        .buildAsyncClient();
+```
+### Uploading logs
+
+For examples on how to upload logs, see the [Examples](#examples) section.
+
+## Key concepts
+
+### Data Collection Endpoint
+
+Data Collection Endpoints (DCEs) allow you to uniquely configure ingestion settings for Azure Monitor. [This 
+article][data_collection_endpoint] provides an overview of data collection endpoints including their contents and structure and how you can create
+and work with them.
+
+### Data Collection Rule
+
+Data collection rules define data collected by Azure Monitor and specify how and where that data should be sent or
+stored. The REST API call must specify a DCR to use. A single DCE can support multiple DCRs, so you can specify a
```

Comment:
it might be useful to explain what DCE is (and maybe add a note that DCR means Data Collection Rule). If you believe everyone reading it should already know it, feel free to ignore :)

Reply:
The section above this introduces Data Collection Endpoints (DCE). I'll also include the acronym for Data collection rule at the beginning of this section.